### ISSUE TITLE
Getting Access to Blob/Tensor reference in jit::script::Module

### ISSUE THREAD

**Issue by soulslicer** [pytorch/pytorch#21165]
## 🚀 Feature

In Caffe, one would be able to set a reference to the blob by doing the following:

`myClass->blob = upCaffeNet->blob_by_name(upImpl->mLastBlobName);`

In Pytorch we have

`std::shared_ptr<torch::jit::script::Module> upTorchNet`

Is there a function I can use for something similar? Right now, there only seems to be the forward pass function in the documentation. Anyway to access or get a reference to the internal blobs? It will be useful for setting previous stages in LSTM's etc. too

**Comment by suo** [comment]
Hi! It depends, what are you trying to reference? If they are parameters (e.g. weights), the `find_parameter` will get you a reference to that tensor. If you are trying to get a submodule, then use `find_module`, a method, then use `find_method`, and so on.

**Comment by soulslicer** [comment]
No, I am trying to get a pointer to the output. If I do model->forward(), I get a torch::Tensor. I would like to a permanent address/memory location of the output

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
